CHANGES.d/20240508_113136_cz_oci_optional_monitoring.md

```diff
@@ -0,0 +1,3 @@
+- Add option to disable OCI container monitoring.
```

Comment:
I'd add the component this change is happening. This kind of ChangeLog will need much more thinking while reading whether someone is impacted by it